preprocessing.py

```diff
@@ -35,7 +36,16 @@ def tokenize_context(context):
 
 
 def word2vec(word2vec_path):
+    # Download word2vec data if it's not present yet
+    if not path.exists(word2vec_path):
```

Comment:
Utility function `get_file` is able to check whether the requested file exists or not.

Reply:
this has nothing to do with get_file, it's the output of glove2word2vec convertion

Reply:
the function name get_word2vec_zip_path might be confusing and incorrect as it returns the file path of unzipped glove file so I updated it in the last commit